### PR TITLE
Improve Storybook configuration

### PR DESCRIPTION
**Pull request by stephenjwatkins**
- Removes unused addons
- Adds accessibility addon
- Brands the Storybook window for EasyPost
- Uses Easy UI breakpoints
- Uses Easy UI grid size
- Update gray tokens to match Figma names (whoops!)

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -1,11 +1,7 @@
 /** @type {import('@storybook/react/types').StorybookConfig} */
 module.exports = {
   stories: ["../easy-ui-react/src/**/*.stories.mdx"],
-  addons: [
-    "@storybook/addon-links",
-    "@storybook/addon-essentials",
-    "@storybook/addon-interactions",
-  ],
+  addons: ["@storybook/addon-essentials", "@storybook/addon-a11y"],
```
Comment (stephenjwatkins) [inline .storybook/main.js:4]: if in the future if we find we want the other addons we can add them them. didn't want to pollute our addons with unused ones

File: easy-ui-tokens/src/color/base.json [hunk 2]
```diff
@@ -51,20 +51,20 @@
...
+      "800": { "value": "#1C222D" },
+      "700": { "value": "#384359" },
+      "600": { "value": "#47547F" },
+      "500": { "value": "#7086B2" },
+      "400": { "value": "#8D9EC1" },
+      "300": { "value": "#A9B6D1" },
+      "200": { "value": "#C6CFE0" },
+      "100": { "value": "#E2E7F0" },
+      "050": { "value": "#F1F1F1" },
       "000": { "value": "#FFFFFF" }
```
Comment (stephenjwatkins) [inline easy-ui-tokens/src/color/base.json:64]: didn't realize they were named particularly in Figma. whoops

File: .storybook/theme.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  barTextColor: colors.text,
+  barSelectedColor: colors.primary,
+  barBg: colors.background,
+
+  // Form colors
+  inputBg: colors.background,
+  inputBorder: colors.border,
+  inputTextColor: colors.text,
+  inputBorderRadius: tokens["shape-border-radius-base"],
+});
```
Comment (stephenjwatkins) [inline .storybook/theme.js:55]: we can change this to be something different in the future per design recs, just wanted to get a structure in place for some kind of identity

**Review by ldewald — APPROVED** [review]
Approving as is but a couple questions for follow up:
* Is it possible to default to the `Docs` view rather than `Canvas`? 
* Is it possible to set the favicon?

**Pull request merged**